### PR TITLE
Make it compile

### PR DESCRIPTION
**Pull request by teintuc**
Juste some quick and dirty fix to make the code compile.

### CODE REVIEW

File: dashlane/vault.go [hunk 1]
```diff
@@ -47,14 +48,21 @@ type Vault struct {
 	Passwords []PasswordEntry `xml:"KWAuthentifiant>KWDataItem"`
 }
 
-func ParseVault(data, password string) (Vault, error) {
+func ParseVault(data, password string) (*Vault, error) {
+	vault := new(Vault)
 	decrypted, err := DecryptVault(data, password)
 	if err != nil {
-		return err
+		return vault, err
```
Comment (masterzen) [inline dashlane/vault.go]: Better return `nil` in case of error

File: dashlane/vault.go [hunk 2]
```diff
@@ -47,14 +48,21 @@ type Vault struct {
...
 	}
 
 	// parse the XML
+ err = xml.Unmarshal([]byte(decrypted), &vault)
```
Comment (masterzen) [inline dashlane/vault.go]: You should run `go fmt` to respect go standards :)

**Review by masterzen — COMMENTED** [review]
Good start :)

File: dashlane/vault.go (hunk 1 above)
Comment (teintuc) [reply dashlane/vault.go]: ok fixed and move the vault declaration

File: dashlane/vault.go (hunk 2 above)
Comment (teintuc) [reply dashlane/vault.go]: I ran go fmt on the project

**Review by masterzen — APPROVED** [review]

**Pull request merged**